### PR TITLE
Increase release version to 1.0.3

### PR DESCRIPTION
**Pull request by javierdelapuente**
Applicable spec: <link>

### Overview

New version of the library with S3, saml and optional integrations to be released.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by nrobinaubertin — APPROVED** [review]

**Review by jdkandersson — APPROVED** [review]

**Comment by jdkandersson** [comment]
One of the tests failed?

**Comment by javierdelapuente** [comment]
> One of the tests failed?

Looks like related to https://github.com/canonical/charm-logdump-action/pull/9, we will have to wait until it is merged.

**Pull request merged**
